src/patternfly/components/Slider/slider.scss

```diff
@@ -1,87 +1,90 @@
-  --#{$slider}__step-tick--BackgroundColor: var(--#{$pf-global}--BorderColor--200);
+  --#{$slider}__step-tick--BackgroundColor: var(--pf-t--global--icon--color--subtle);
   --#{$slider}__step-tick--TranslateX: -50%;
-  --#{$slider}__step-tick--BorderRadius: var(--#{$pf-global}--BorderRadius--lg);
-  --#{$slider}__step--m-active__slider-tick--BackgroundColor: var(--#{$pf-global}--primary-color--100);
+  --#{$slider}__step-tick--BorderRadius: var(--pf-t--global--border--radius--large);
+  --#{$slider}__step--m-active__slider-tick--BackgroundColor: var(--pf-t--global--icon--color--brand--clicked);
   --#{$slider}__step--first-child__step-tick--TranslateX: 0;
   --#{$slider}__step--last-child__step-tick--TranslateX: -100%;
-
+  
   // step label
   --#{$slider}__step-label--TranslateX: -50%;
-  --#{$slider}__step-label--Top: calc(var(--#{$pf-global}--spacer--xl) + var(--#{$slider}__rail-track--Height));
+  --#{$slider}__step-label--Top: calc(var(--pf-t--global--spacer--xl) + var(--#{$slider}__rail-track--Height));
   --#{$slider}__step--first-child__step-label--TranslateX: 0;
   --#{$slider}__step--last-child__step-label--TranslateX: -100%;
-
+  
   // thumb
-  --#{$slider}__thumb--Top: calc(var(--#{$slider}__rail-track--Height) / 2 + var(--#{$pf-global}--spacer--md));
+  --#{$slider}__thumb--Top: calc(var(--#{$slider}__rail-track--Height) / 2 + var(--pf-t--global--spacer--md));
   --#{$slider}__thumb--Width: #{pf-size-prem(16px)};
   --#{$slider}__thumb--Height: #{pf-size-prem(16px)};
   --#{$slider}__thumb--Left: var(--#{$slider}--value);
-  --#{$slider}__thumb--BackgroundColor: var(--#{$pf-global}--primary-color--100);
+  --#{$slider}__thumb--BackgroundColor: var(--pf-t--global--color--brand--default);
   --#{$slider}__thumb--TranslateX: -50%;
   --#{$slider}__thumb--TranslateY: -50%;
-  --#{$slider}__thumb--BorderRadius: var(--#{$pf-global}--BorderRadius--lg);
+  --#{$slider}__thumb--BorderRadius: var(--pf-t--global--border--radius--large);
   --#{$slider}__thumb--BoxShadow--base:
-    0 0 0 2px var(--#{$pf-global}--BackgroundColor--100),
-    0 0 0 3px var(--#{$pf-global}--primary-color--100);
+    0 0 0 2px var(--pf-t--global--background--color--primary--default),
+    0 0 0 3px var(--pf-t--global--color--brand--default);
   --#{$slider}__thumb--hover--BoxShadow: var(--#{$slider}__thumb--BoxShadow--base);
   --#{$slider}__thumb--focus--BoxShadow: var(--#{$slider}__thumb--BoxShadow--base);
   --#{$slider}__thumb--active--BoxShadow:
     var(--#{$slider}__thumb--BoxShadow--base),
-    0 0 2px 5px var(--#{$pf-global}--active-color--200);
-  --#{$slider}__thumb--before--Width: var(--#{$pf-global}--target-size--MinWidth);
-  --#{$slider}__thumb--before--Height: var(--#{$pf-global}--target-size--MinHeight);
-  --#{$slider}__thumb--before--BorderRadius: var(--#{$pf-global}--BorderRadius--lg);
+    0 0 2px 5px var(--pf-t--global--color--nonstatus--blue--clicked);
+  --#{$slider}__thumb--before--Width: 44px;
```

Comment:
My only question is whether we want to keep using the global target size here (maybe just for now) and I would guess we might want to make this a dimension token in the future?